src/resources/views/crud/buttons/on_the_fly/create.blade.php

```diff
@@ -0,0 +1,11 @@
+<span class="browse btn btn-sm btn-light move" id="{{ $onTheFlyEntity }}-on-the-fly-create-{{$name}}" type="button"><span class="fa fa-plus"></span> New</span>
```

Comment:
I'm not sure this button has its place inside the ```crud/buttons``` folder. It's something that only this one field type uses. So maybe a better place for it would be inside ```crud/fields/inc```?